cosmos/providers/dbt/parser/project.py

```diff
@@ -132,7 +132,7 @@ def __post_init__(self) -> None:
         for base_node in ast.find_all(jinja2.nodes.Call):
             if hasattr(base_node.node, "name"):
                 # check we have a ref - this indicates a dependency
-                if base_node.node.name == "ref" or base_node.node.name == "source":
+                if base_node.node.name == "ref":
```

Comment:
It seems "ref" was introduced as part of the Open Lineage support implementation:
https://github.com/astronomer/astronomer-cosmos/commit/8df9ee47f18753b281b200dd7a847824d042b2f7

@chrishronek @jlaneve, any thoughts about this change?

Reply:
this shouldn't be required by open lineage!